### DISCUSSION TITLE
How to perform a perfect paradrop without getting hurt?

### DISCUSSION THREAD

**Discussion by acknowledge14** [acemod/ACE3#9209] [Q&A] [answered]
With ACE medical enabled, of course. When I open the vanilla parashute then touch the ground, the wounds are all over my body and limbs, I must get banaged several times to stop the bleed.

So is there any way to avoid these injuries when touching the ground?

**Comment by Andx667** [comment]
You must be very slow - less then 5 km/h - when landing. Use an altimeter provided by ace to see your height and speed.

**Reply by Fulgurofist** [reply]
il faut également avant de toucher me sol, d'appuyer sur la touche "s" par défaut, pour amortir et effectuer un flaire

**Reply by rautamiekka** [reply, marked as answer]
\+ open the chute at 200 meters at the latest.

Steering in a circle will also help avoid getting hurt, but it's more useful for getting your bearings.
